### PR TITLE
First iteration of the documentation

### PR DESCRIPTION
**Pull request by ishuverma**

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+This demo uses the CDK platform: learn how to install CDK https://github.com/redhatdemocentral/cdk-install-demo[here]
+
+*Please note: This repository is currently under development and the feature set/components are subject to change without notice*
+
+== Components
+
+* [AMQ and MQTT](https://access.redhat.com/documentation/en/red-hat-xpaas/0/paged/red-hat-xpaas-a-mq-image/)
+* [Realtime Decision Server](https://access.redhat.com/documentation/en/red-hat-xpaas/0/paged/red-hat-xpaas-a-mq-image/)
+* [Fuse Integration Services](https://access.redhat.com/documentation/en/red-hat-xpaas/version-0/red-hat-xpaas-fuse-integration-services-image/)
```
Comment (sabre1041) [inline README.adoc:20]: Links are not in asciidoc format

File: README.adoc [hunk 2]
```diff
@@ -0,0 +1,61 @@
+= IoT OpenShift Demo
+
+:Author:    Andrew Block, Ishu Verma
+:Email:     ablock@redhat.com, iverma@redhat.com
+:Date:      10/10/2016
+
+== Overview
+Demonstration of Intelligent IoT Gateway on the Red Hat OpenShift Container Platform. The Gateway enables IoT data transformation, routing and real time decision-making at the edge. OpenShift provides secure enterprise-grade container platform to accelerate IoT application delivery. Each component of the demo runs in its own container, thereby, allowing security, scalability and ease of deployment.
+
+image:images/ose-iot.png[]
```
Comment (sabre1041) [inline README.adoc:10]: Do we want to denote these are not full instances of BRMS and Fuse and instead xPaaS?

**Review by sabre1041 — CHANGES_REQUESTED** [review]
Love the diagram and the move to .adoc format

It appears you are missing the most recent version of the README that includes how to set up Zeppelin. Would you be able to integrate those changes in?

**Comment by ishuverma** [comment]
Yes, I noticed the changes yesterday midway. I'll rebase to the new rev and
create a new pull request.

I've lot of other screenshots and stuff I want to add.

ishu

**Comment by ishuverma** [comment]
sent another request based on the latest rev of readme

**Pull request closed without merging**